codersdk/workspaces.go

```diff
@@ -107,6 +107,8 @@ type CreateWorkspaceBuildRequest struct {
 
 	// Log level changes the default logging verbosity of a provider ("info" if empty).
 	LogLevel ProvisionerLogLevel `json:"log_level,omitempty" validate:"omitempty,oneof=debug"`
+	// TemplateVersionPresetID is the ID of the template version preset to use for the build.
```

Comment:
Nit: this comment is a bit redundant.

```suggestion
	// TemplateVersionPresetID is the ID of the selected preset, if available.
```